test/functional.js

```diff
@@ -227,6 +227,35 @@ describe('Functional tests using webpack', function() {
             });
         });
 
+        it('Deploying to an unknown (at compile-time) subdirectory is no problem', (done) => {
+            const config = createWebpackConfig('public/build', 'dev');
+            config.addEntry('main', './js/code_splitting');
```

Comment:
I think we should also include an entry that loads a CSS file normally (i.e. without code splitting) that includes a path to an image or a font. Then, after running webpack, we can manually check the contents of the final CSS file to make sure that the path to that image is correct. As I understand it, THAT is the biggest problem we're having with relative paths: the paths to images/fonts in CSS files was being generated incorrectly.

Unfortunately, the `assertResourcesLoadedCorrectly` method you're using below works for JS only - that's why we need to test the contents of the file.